.github/workflows/deploy-2-start.yml

```diff
@@ -73,7 +73,7 @@ jobs:
           spack env activate ${{ inputs.env-name }}
           spack --debug install --fresh || exit $?
           spack module tcl refresh --delete-tree -y
-          spack find --paths > ${{ vars.SPACK_LOCATION }}/var/spack/environments/${{ inputs.env-name }}/spack.location
+          spack find --format '{name}@{version} {prefix}' | jq -Rn '[inputs | split(" ") | {(.[0]):(.[1])}] | add' > ${{ vars.SPACK_LOCATION }}/var/spack/environments/${{ inputs.env-name }}/spack.location
```

Comment:
So what happens if there is normal text and `json` formatted text appended? Is that parseable?

Reply:
One is `spack.location` and one is `spack.location.json` - they won't be appended together:
https://github.com/ACCESS-NRI/build-cd/pull/52/files#diff-8984338bca7c293efa309256806ef1822893d031812c36f28b661d9f4b142383R76-R77